frontend/views/containers/group-settings/InvitationLinkModal.vue

```diff
@@ -39,14 +54,17 @@ export default ({
     },
     welcomeInviteSecret () {
       const key = this.$store.state.secretKeys[this.welcomeInviteId]
-      if (typeof key !== 'string') {
-        return serializeKey(key, true)
-      } else {
-        return key
+      if (!key) {
+        console.error(`undefined key for welcomeInviteId: ${this.welcomeInviteId}`)
```

Comment:
I'd say 'missing' or 'not found' rather than undefined to keep in line with the rest of the terminology used about keys.

Reply:
OK feel free to update in your branch, I'll go ahead and merge since approved